packages/components/stories/List.js

```diff
@@ -600,4 +600,34 @@ storiesOf('List', module)
 				<List {...tprops} />
 			</div>
 		);
+	})
+	.add('Virtualized with filter', () => {
+		const dockedProps = Immutable.fromJS(props).toJS();
+		dockedProps.list.items = [dockedProps.list.items[0]];
+		dockedProps.toolbar.actionBar = null;
+
+		const inputProps = Immutable.fromJS(dockedProps).toJS();
+		inputProps.toolbar.filter.docked = false;
+
+		const highlightedProps = Immutable.fromJS(inputProps).toJS();
+		highlightedProps.toolbar.filter.highlight = true;
+
+		const inputDebounceProps = Immutable.fromJS(inputProps).toJS();
+		inputDebounceProps.toolbar.filter.debounceTimeout = 300;
+
+		return (<div style={{ height: '60vh' }} className="virtualized-list">
+			<IconsProvider />
+
+			<h1>List</h1>
+			<h2>Definition</h2>
+			<p>Toolbar Filter</p>
+			<h2>Docked</h2>
+			<List {...dockedProps} virtualized />
```

Comment:
To avoid the list to extends its size, let's wrap each list in a controlled size div